sources/HUBViewController.m

```diff
@@ -441,6 +441,11 @@ - (void)scrollToComponentOfType:(HUBComponentType)componentType
                                  completion:completion];
 }
 
+-(void)reload
+{
+    [(HUBViewModelLoaderImplementation*)self.viewModelLoader loadViewModelRegardlessOfReloadPolicy];
```

Comment:
Instead of casting, can you update the `viewModelLoader` property to be typed as `HUBViewModelLoaderImplementation *`? (It's already that type in the initializer)